lib/version.js

```diff
@@ -0,0 +1,108 @@
+var /**
+     * @const
+     * @type {String[]}
+     */
+    PACKAGES_TO_CONSIDER = [
+        'chai-postman',
+        'postman-collection',
+        'postman-request',
+        'postman-sandbox',
+        'uniscope',
+        'uvm'
+    ],
+
+    /**
+     * Generate dependency tree of runtime.
+     *
+     * @param {String} [dependencyName] - Module name
+     * @param {Boolean} [findDeepDependencies=true] - Do depth dependencies traversal or stop at root
+     * @param {Object} [dependencyTree={}] - Object to store dependencies
+     * @returns {Object}
+     *
+     * @note Don't use tree argument in this function, that is for internal use only.
+     *
+     * @example <caption>Returned Object Structure</caption>
+     * {
+     *     version: '7.6.1',
+     *     dependencies: {
+     *         'postman-collection' : {
+     *             version: '3.4.2',
+     *             dependencies: {
+     *                 'postman-request': {
+     *                     version: '2.88.1-postman.5'
+     *                 }
+     *             }
+     *         },
+     *         'postman-request': {
+     *             version: '2.88.1-postman.5'
+     *         }
+     *     }
+     * }
+     */
+    dependenciesTree = function (dependencyName, findDeepDependencies, dependencyTree) {
+        !dependencyTree && (dependencyTree = {});
+        (findDeepDependencies === undefined) && (findDeepDependencies = true);
+
+        var version,
+            // runtime's package.json
+            packageJson = require('../package.json'),
```

Comment:
This can be moved to top as constant since this statement is not required for every recursive call.

Reply:
Done.